src/sql/workbench/contrib/notebook/browser/notebookViews/notebookViews.component.ts

```diff
@@ -251,6 +268,25 @@ export class NotebookViewComponent extends AngularDisposable implements INoteboo
 
 		let viewOptions = this._instantiationService.createInstance(ViewSettingsAction, this.views);
 
+		let viewsContainer = document.createElement('li');
```

Comment:
Make sure you're going through all the UI elements you're adding and test for a11y usability - that things have proper names for screen readers, keyboard navigation works as expected, etc.

Reply:
I'll look into this.